middleware/cache/README.md

```diff
@@ -60,3 +60,13 @@ Proxy to Google Public DNS and only cache responses for example.org (or below).
 proxy . 8.8.8.8:53
 cache example.org
 ~~~
+
+### TTL examples  
+no cache -> record TTL is used  
+*cache* -> cached with a TTL of max 3600 or 1800 - for smaller record provided TTLs the smaller one is used  
+*cache 0* -> cached with a TTL of 5  
+*cache 300* -> cached with a TTL of max 300 - for smaller record provided TTLs the smaller one  
+
+## Interaction with other middlewares
+
+The *cache* middleware does cache all answers except for metadata and transfer queries. No cache invalidation is available from other middlewares. Therefore the cache might serve stale records for reloaded zones until the cache expires.
```

Comment:
The first sentence is a repeat of something said above.

The point that, I think, you want to make is that there is currently no mechanism to invalidate a cache entry. This can be explained in the first paragraph.

I also think it is not fair to say "stale records", as they are perfectly valid because that's what a TTL allows you to do.